src/components/data-table-v2/data-table-v2-expandable.html

```diff
@@ -8,8 +9,8 @@
         </svg>
         <label id="search-input-label-10" class="bx--label" for="search__input-10">Search</label>
         <input class="bx--search-input" type="text" id="search__input-10" role="search" placeholder="Search" aria-labelledby="search-input-label-10">
-        <svg class="bx--search-close bx--search-close--hidden" width="16" height="16" viewBox="0 0 16 16" fill-rule="evenodd">
-          <path d="M8 0C3.6 0 0 3.6 0 8s3.6 8 8 8 8-3.6 8-8-3.6-8-8-8zm3.5 10.1l-1.4 1.4L8 9.4l-2.1 2.1-1.4-1.4L6.6 8 4.5 5.9l1.4-1.4L8 6.6l2.1-2.1 1.4 1.4L9.4 8l2.1 2.1z"></path>
+        <svg class="bx--search-close bx--search-close--hidden" width='10' height='10' viewBox='0 0 10 10' fill-rule='evenodd'>
```

Comment:
Is the quote change intentional here? 😮

Reply:
@joshblack Hmm, this is the code copy / pasted from the website. We should make sure the copy from the website has double quotes

cc @marijohannessen